### ISSUE TITLE
Add skip once feature for recurring alarms

### ISSUE THREAD

**Issue by Mcmartelle** [SimpleMobileTools/Simple-Clock#36]
Sometimes I get up before an alarm and I don't want it to go off, since I'm already up. Currently I can: 1) Just leave it on and get interrupted while I'm already awake. 2) Turn the whole recurring alarm off and risk forgetting to turn it on again before the next morning. I'd like a "skip once" option where the next instance of the recurring alarm won't sound, but all of later instances will.

**Comment by abacao** [comment]
Yes please implement skip next alarm or next X times in recurring alarm. It's really one of the options I miss the most.

**Comment by Pablohn26** [comment]
Xiaomi clock implement this feature allowing the user to select how do you want to turn off the alarm.
It gives you two options:
* Turn off just once (today)
* Turn off permanently

![IMG_20200604_002923](https://user-images.githubusercontent.com/1228418/83695649-c5f81b80-a5fa-11ea-91a3-04040ff7fb4c.jpg)
the picture is in Spanish, sorry. But I hope you understand it

**Comment by guleriaakshit** [comment]
> Sometimes I get up before an alarm and I don't want it to go off, since I'm already up. Currently I can: 1) Just leave it on and get interrupted while I'm already awake. 2) Turn the whole recurring alarm off and risk forgetting to turn it on again before the next morning. I'd like a "skip once" option where the next instance of the recurring alarm won't sound, but all of later instances will.

I second this 😊
